.gitmodules

```diff
@@ -0,0 +1,16 @@
+[submodule "linux"]
+	path = linux
+	url = git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+	shallow = true
+[submodule "llvm"]
+	path = llvm
+	url = https://git.llvm.org/git/llvm.git/
+	shallow = true
+[submodule "clang"]
+	path = clang
+	url = https://git.llvm.org/git/clang.git/
+	shallow = true
+[submodule "buildroot"]
+	path = buildroot
+	url = git://git.buildroot.net/buildroot
+	shallow = true
```

Comment:
interesting, if I try to clone this locally, I get an error that the buildroot server doesn't advertise the commit listed.  travis ci doesn't see this because it always clones with --depth 50.  I wonder if I should remove the shallow clone here, or update the readme on how to initialize the repo?

Reply:
removed buildroot as a submodule.  I *think* this won't be an issue with the kernel if we wire up some kind of auto update.

Reply:
I've added 4eba2e8bbcc86cfdf4db7abf7f7cb948be1567cb + added a [Travis cron](https://docs.travis-ci.com/user/cron-jobs/) for now but I could wire up a cron job on my server to automatically bump and commit the update every couple of hours like I am doing for the other CBL repos.